### PR TITLE
Api gateway stage

### PR DESCRIPTION
**Pull request by albinekb**

### CODE REVIEW

File: lib/tasks.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ exports.parseOptions = {
       }
     }
 
+    if (!ctx.args['--api-gateway-stage']) {
```
Comment (LinusU) [inline lib/tasks.js]: Remove the `if` 😄

File: lib/tasks.js [hunk 2]
```diff
@@ -173,11 +175,13 @@ exports.updateApiGateway = {
...
 exports.deployApi = {
   title: 'Deploying the API to a live address',
-  enabled: (ctx) => Boolean(ctx.args['--deploy-to']),
+  enabled: (ctx) => Boolean(ctx.apiGatewayStage),
   task: async (ctx, task) => {
-    const stage = ctx.args['--deploy-to']
+    const stage = ctx.apiGatewayStage
     const { region } = parseArn(ctx.lambdaArn)
 
+    task.title = `Deploying the API to API gateway stage: ${stage}, region: ${region}`
```
Comment (LinusU) [inline lib/tasks.js]: `region` is undefined here?

Comment (LinusU) [reply lib/tasks.js]: hahaha, I'm blind 😂

Comment (albinekb) [reply lib/tasks.js]: I just added it because why not 😄

**Comment by LinusU** [comment]
Thanks!

**Pull request merged**